releasenotes/notes/optimize-ksm-configmap-collection-1e3eb7425a7af050.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    The `kubernetes_state_core` check now collects only metadata for configmaps,
+    reducing memory usage in the DCA while preserving full metric functionality.
```

Comment:
This might be a dumb question, but what is the DCA? Consider spelling out the acronym if it wouldn't be common knowledge for the audience of the release note. 😄

Reply:
Datadog Cluster Agent, switched to Cluster Agent for clarity